### DISCUSSION TITLE
install error: Timeout was reached: [api.github.com] Failed to connect to api.github.com port 443 after 10000 ms:

### DISCUSSION THREAD

**Discussion by aura6666** [MRCIEU/TwoSampleMR#446] [General]
Whether I choose to install online or locally, an error timeout is reported. 
 I have changed a lot of network environments or VPN. 
 Has anyone encountered this situation and resolved it? 

![image](https://github.com/MRCIEU/TwoSampleMR/assets/148430552/6c5072be-7276-404d-888a-757f08191be0)

> remotes::install_github("MRCIEU/TwoSampleMR")
error: Failed to install 'unknown package' from GitHub:
  Timeout was reached: [api.github.com] Failed to connect to api.github.com port 443 after 10001 ms: Timeout was reached
> devtools::install_local("D:/R/TwoSampleMR.zip")
error: Failed to install 'TwoSampleMR.zip' from local:
  Timeout was reached: [api.github.com] Failed to connect to api.github.com port 443 after 10000 ms: Timeout was reached

**Comment by remlapmot** [comment]
This looks like some problem with your internet connection or with the availability of GitHub in your area, or maybe you hit your GitHub rate limit (too many downloads from GitHub in a certain time limit).

It looks like you are on a Windows machine, so you can install a prebuilt binary version of the package from our mrcieu.r-universe with the following code

```r
install.packages('TwoSampleMR', repos = c('https://mrcieu.r-universe.dev', 'https://cloud.r-project.org'))
```

**Reply by aura6666** [reply]
Resolved.
Thank you from the bottom of my heart.

**Reply by remlapmot** [reply]
That's great
